modules/pruning-images.adoc

```diff
@@ -5,107 +5,89 @@
-initiated.
-
-.^|`--confirm`
-|Indicate that pruning should occur, instead of performing a dry-run. This
-requires a valid route to the integrated container image registry. If this
-command is run outside of the cluster network, the route must be provided
-using `--registry-url`.
-
-.^|`--force-insecure`
-|Use caution with this option. Allow an insecure connection to the container
-registry that is hosted via HTTP or has an invalid HTTPS certificate.
-
-.^|`--keep-tag-revisions=<N>`
-|For each imagestream, keep up to at most `N` image revisions per tag (default
-`3`).
-
-.^|`--keep-younger-than=<duration>`
-|Do not prune any image that is younger than `<duration>` relative to the
-current time. Do not prune any image that is referenced by any other object that
-is younger than `<duration>` relative to the current time (default `60m`).
-
-.^|`--prune-over-size-limit`
-|Prune each image that exceeds the smallest limit defined in the same project.
-This flag cannot be combined with `--keep-tag-revisions` nor
-`--keep-younger-than`.
-
-.^|`--registry-url`
-|The address to use when contacting the registry. The command attempts to use a
-cluster-internal URL determined from managed images and imagestreams. In case
-it fails (the registry cannot be resolved or reached), an alternative route that
-works needs to be provided using this flag. The registry host name can be
-prefixed by `https://` or `http://` which enforces particular connection
-protocol.
-
-.^|`--prune-registry`
-|In conjunction with the conditions stipulated by the other options, this option
-controls whether the data in the registry corresponding to the {product-title}
-image API object is pruned. By default, image pruning processes both the image
-API objects and corresponding data in the registry. This options is useful when
-you are only concerned with removing etcd content, possibly to reduce the number
-of image objects (but are not concerned with cleaning up registry storage) or
-intend to do that separately by hard pruning the registry, possibly during an
-appropriate maintenance window for the registry.
-|===
 
+[IMPORTANT]
+====
+The Image Registry Operator's behavior for managing the pruner is orthogonal to the `ManagementState` specified on the Image Registry Operator's `ClusterOperator` object. If the image registry operator is not in the `Managed` state, the image pruner can still be configured and managed by the pruning custom resource.
+
+However, the `ManagementState` of the Image Registry Operator alters the behavior of the deployed image pruner job:
+
+* `Managed`: the `--prune-registry` flag for the image pruner is set to `true`.
+* `Removed`: the `--prune-registry` flag for the image pruner is set to `false`, meaning it only prunes image metatdata in etcd.
+* `Unmanaged`: the `--prune-registry` flag for the image pruner is set to `false`.
+====
+
+
+////
 [id="pruning-images-conditions_{context}"]
```

Comment:
The following content is currently commented out. Is it still relevant? If not, can I go ahead and remove it?